### DISCUSSION TITLE
Future of AIR and big Thanks to Andrew Frost and his Team

### DISCUSSION THREAD

**Discussion by Marcophono2** [airsdk/Adobe-Runtime-Support#610] [General]
I admit: This isn't really an issue. It is a very big THANK YOU to Andrew and his team! They made the unbelievable possible and turned Adobe's neglected muck heap into a serious programming environment again in record time! I cannot comprehend how engaged and fast the workflow now is in the AIR development. I have big fun again to program in Actionscript and hope that Andrew will get all support from Harman he requests. He also deserves it to have a break from time to time which he seems never to take. 

Thank you all, have a Merry Christmas and a Happy New Year!
Marc 🎄 ☃️ 🎆

**Comment by ajwfrost** [comment]
Thank you for that, I've forwarded the message to the team. It's been a slightly crazy second half of the year with us getting overwhelmed with requests for extended Flash Player support, so I'm conscious we've not actually had the full focus of the team being on AIR and there are an increasing number of issues with the new Mac platforms that we need to get on top of :-(

But messages like this make it all worthwhile .. we'll continue to try to improve things, and to get ahead of the game with some of these other updates, and look forward to adding some of the features and fixes that people are requesting. I think 2021 will start to see some better improvements as we try to take this platform forward.

Happy Christmas everyone!

**Comment by gene-pavlovsky** [comment]
@ajwfrost Can you please post some news on Harman's AIR website? The last update seems to be from 2019. It would be interesting to know what's the current status - what had been done already, and also what are the plans for next year.
I'm about to restart support / modernize an old AIR project for a client of mine, and it'd be great to know what kind of goodies we can get by getting a license and switching to the latest Harman SDK.
Thanks :)
